src/Common/tests/System.Xml.XPath/CoreFunctionLibrary/StringFunctionsTests.cs

```diff
@@ -529,6 +529,34 @@ public static void StringFunctionsTest2436()
 
         /// <summary>
         /// Verify result.
+        /// normalize-space("") = ""
+        /// </summary>
+        [Fact]
+        public static void StringFunctionsTest2473()
```

Comment:
The existing translate function doesn't handle surrogate pairs. For example the following will fail as is operates on single chars.

    var fourOClock = char.ConvertFromUtf32(0x1F553);
    var fiveOClock = char.ConvertFromUtf32(0x1F554);
    var testExpression = @"translate(""" + fourOClock + fiveOClock + @""", """ + fourOClock + @""", """")";
    var expected = fiveOClock;